book/controller.rst

```diff
@@ -794,6 +794,22 @@ Just like when creating a controller for a route, the order of the arguments of
 order of the arguments, Symfony will still pass the correct value to each
 variable.
 
+Checking the Validity of a CSRF Token
+-------------------------------------
+
+Sometimes you want to use CSRF protection in an action where you don't want to use a
+Symfony form.
+
+If, for example, you're doing a DELETE action, you can use the
+:method:`Symfony\\Component\\Form\\Extension\\Csrf\\CsrfProvider\\CsrfProviderAdapter::isTokenValid`
+method to check the CSRF token::
+
+    use Symfony\Component\Security\Csrf\CsrfToken;
```

Comment:
Thanks, will change this to use `generateCsrfToken` from `Symfony\Component\Form\Extension\Csrf\CsrfProvider`.